### DISCUSSION TITLE
TFT_eSPI with ILI9488

### DISCUSSION THREAD

**Discussion by joanmor** [Bodmer/TFT_eSPI#2794] [Compatible support libraries]
I am doing a project in which I connect my arduino mega board to a 4" ILI9488 SPI tft.
I can't get it to work with the TFT_eSPI library.
I know that the problem is not electronic since with simpler libraries it works.
Regarding TFT_eSPI I have made the following modifications to adapt it:
![image](https://github.com/Bodmer/TFT_eSPI/assets/142510951/94101169-f7c1-4537-884c-671ffc0963aa)
![image](https://github.com/Bodmer/TFT_eSPI/assets/142510951/65eb5d56-3967-4cf8-be59-23688467a10b)
Does anyone know if I need to modify more things or what is the problem?

**Comment by Bodmer** [comment]
Unfortunately the library is not appropriate for the Mega as it is intended for 32 bit processors that have lots of RAM.

You may get lucky if you change the PIN_Dx designation to Dx (the PIN_Dx designation were intended for ESP8266 only due to pin mapping variations between vendors)

**Reply by joanmor** [reply]
How can I do this?

**Reply by Bodmer** [reply]
Example:
#define TFT_CS   D10
